src/Files.App/Data/Commands/Manager/CommandCodes.cs

```diff
@@ -186,10 +186,13 @@ public enum CommandCodes
 		PreviousTab,
 		NextTab,
 		CloseSelectedTab,
+
+		// Shell Panes
 		OpenNewPane,
 		CloseActivePane,
 		FocusLeftPane,
 		FocusRightPane,
+		SplitPane,
```

Comment:
I think we'll need an action for horizontal and vertical.

Reply:
Done.